x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/jdbc/PreparedStatementTestCase.java

```diff
@@ -66,13 +103,70 @@ public void testSupportedTypes() throws Exception {
                     assertEquals(byteVal, results.getByte(8));
                     assertEquals(shortVal, results.getShort(9));
                     assertEquals(bigDecimalVal, results.getBigDecimal(10));
+                    assertEquals(timestampVal, results.getTimestamp(11));
+                    assertEquals(timestampValWithCal, results.getTimestamp(12));
+                    assertEquals(dateVal, results.getDate(13));
+                    assertEquals(dateValWithCal, results.getDate(14));
+                    assertEquals(timeVal, results.getTime(15));
+                    assertEquals(timeValWithCal, results.getTime(16));
+                    assertEquals(new Timestamp(calendarVal.getTimeInMillis()), results.getObject(17));
+                    assertEquals(timestampVal, results.getObject(18));
+                    assertEquals(timestampVal, results.getObject(19));
+                    assertFalse(results.next());
+                }
+            }
+        }
+    }
+
+    public void testDateTime() throws IOException, SQLException {
```

Comment:
Can you please break this method down into 3 different tests? This way if one of them breaks, it will be easier to identify the test that did it.